forgettable-frontend/src/pages/home/Home.js

```diff
@@ -152,7 +152,7 @@ function Home() {
           {encountersList.map((encounter, index) => {
             return (
               <div key={index + 'e'} className={classes.home_cardWrapper} onMouseEnter={(event) => handleEncounterHover(event, index)}>
-                <Link to={`/encounters/${encounter._id}`} style={{textDecoration: 'none'}}>
+                <Link to={`/encounters`} style={{textDecoration: 'none'}}>
```

Comment:
Oooh I assume this is removed because we now use modals right?

Reply:
yup! i figured until we pass in a prop thats like "have this modal open" i would just direct to the list page. thanks for the reminder to make a issue haha

Reply:
No worries! You're always one step ahead :D